index.html

```diff
@@ -4409,6 +4438,122 @@ <h3 id="looping-AudioBufferSourceNode">
         </section>
       </section>
       <section>
+        <h2 id="ConstantSourceNode">
+          The ConstantSourceNode Interface
+        </h2>
+        <p>
+          This interface represents a constant audio source whose output is
+          nominally a constant value. It is useful as a constant source node in
+          general and can be used as if it were a constructible
+          <a><code>AudioParam</code></a> by automating its
+          <a><code>sourceValue</code></a> or connecting another node to it.
+        </p>
+        <p>
+          The single output of this node consists of one channel (mono).
+        </p>
+        <pre>
+  numberOfInputs  : 0
+  numberOfOutputs : 1
+        </pre>
+        <dl title=
+        "[Constructor(BaseAudioContext context, optional ConstantSourceOptions options)]interface ConstantSourceNode : AudioNode"
+        class="idl">
+          <dt>
+            readonly attribute AudioParam sourceValue
+          </dt>
+          <dd>
+            <p>
+              The constant value of the source. Its default value is 0. This
```

Comment:
I think both are good defaults; 1 is better if you think the node is a constant source; 0 is slightly better if you think it's an audioparam that might be connected to some source.